internal-export-file/export-report-pdf/src/export-report-pdf.py

```diff
@@ -509,6 +515,363 @@ def _process_threat_actor_individual(self, entity_id, file_name):
             entity_id, file_name, pdf_contents, "application/pdf"
         )
 
+    def _process_case_incident(self, entity_id, file_name):
+        """
+        Process a Case Incident entity and upload as pdf.
+        """
+        # Get the Case
+        case_dict = self.helper.api_impersonate.case_incident.read(id=entity_id)
+        content_query = '{case (id:"' + entity_id + '") {content}}'
```

Comment:
I think we could mutualize some code of this function, to have a _process_case that can be called inside _process_case_rfi, _process_case_incident, _process_case_rft. What are the differences between each of these functions ?

Reply:
even the templates look exactly the same

Reply:
I'm trying to change this code while keeping it readable

Reply:
Done :)

Reply:
Perfect :) and what about the templates ? Do you think we could have the same template for all cases types ?

Reply:
I'm trying to find a solution too, there are few differences... it's true

Reply:
Done too